README.md

```diff
@@ -13,12 +13,12 @@ processes, goals, scope, and deliverables.
 * [Security design document](design/security.md)
 * [Specification](docs/wasi-crypto.md)
 * Interface definitions:
-  * common types and functions ([witx](witx/codegen/proposal_common.witx), [doc](witx/codegen/proposal_common.md))
-  * symmetric operations ([witx](witx/codegen/proposal_symmetric.witx), [doc](witx/codegen/proposal_symmetric.md))
-  * common types and functions for asymmetric operations ([witx](witx/codegen/proposal_asymmetric_common.witx), [doc](witx/codegen/proposal_asymmetric_common.md))
-  * signatures ([witx](witx/codegen/proposal_signatures.witx), [doc](witx/codegen/proposal_signatures.md))
-  * key exchange ([witx](witx/codegen/proposal_kx.witx), [doc](witx/codegen/proposal_kx.md))
-  * external secrets ([witx](witx/codegen/proposal_external_secrets.witx), [doc](witx/codegen/proposal_external_secrets.md))
+  * common types and functions ([witx](witx/proposal_common.witx), [doc](witx/proposal_common.md))
```

Comment:
Should I use file in codegen replace?

Reply:
The docs are in `codegen/` so we should keep these :)